.env.example

```
@@ -0,0 +1,39 @@
+MEMORY_BASE_URL=http://host.docker.internal:7912
+TTS_BASE_URL=https://supertoneapi.com
+TTS_API_KEY=YOUR_TTS_API_KEY
+
+# JWT Configuration (IMPORTANT: Use a secure random key in production)
+JWT_SECRET_KEY=YOUR_JWT_SECRET_KEY_MINIMUM_32_CHARACTERS
+JWT_ACCESS_TOKEN_LIFETIME_MINUTES=15
+JWT_REFRESH_TOKEN_LIFETIME_DAYS=30
+
+# OAuth2 Configuration
+OAUTH2_ENABLED=true
+GOOGLE_OAUTH_ENABLED=true
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The GOOGLE_OAUTH_ENABLED key should go before the OAUTH2_ENABLED key

(UnorderedKey)